packages/commonwealth/client/scripts/views/components/proposals/voting_actions.tsx

```diff
@@ -70,7 +72,7 @@ export const VotingActions = (props: VotingActionsProps) => {
   }
 
   const emitRedraw = () => {
-    app.proposalEmitter.emit('redraw');
+    redrawProposals({});
```

Comment:
This will work the first time only. If the state value is `{}`, then setting it again to `{}` won't cause a re-render. Let's use Math.random() or some other random value

Reply:
Good catch, will update.

Reply:
Please see 571bfe0c44615b84c2e50d4ff3e45c9be1f39ad1 and ensure it meets this ask's criterion. @mzparacha

Reply:
that works